skin_generator/generator.cpp

```diff
@@ -99,14 +99,14 @@ void SkinGenerator::ProcessSymbols(std::string const & svgDataDir,
           QSize size = svgSize * (symbolSize.width() / 18.0); // 18.0 is the size of the -m SVG files
 
           // Fitting symbol into symbolSize, saving aspect ratio.
-          if (size.width() > symbolSize.width())
+          if (size.width() > symbolSize.width() * 24.0 / 18.0)
```

Comment:
Could you please add a code comment explaining why there is a "* 24 / 18"?

Reply:
Added comment

Reply:
I have just refactored this to make it clearer what is happening here